examples/analog_out/cont_gen_voltage_wfm_int_clk.py

```diff
@@ -0,0 +1,45 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to output a continuous periodic
+waveform using an internal sample clock.
+"""
+
+import numpy as np
+import numpy.typing
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+
+def create_sine_wave(
+    frequency: float, amplitude: float, sampling_rate: float, duration: float
+) -> numpy.typing.NDArray[numpy.double]:
+    """Generate a sine wave."""
+    t = np.arange(0, duration, 1 / sampling_rate)
+
+    return amplitude * np.sin(2 * np.pi * frequency * t)
+
+
+def main():
+    """Continuously generates a sine wave."""
+    with nidaqmx.Task() as task:
+        sampling_rate = 1000.0
+        task.ao_channels.add_ao_voltage_chan("Dev1/ao0")
+        task.timing.cfg_samp_clk_timing(sampling_rate, sample_mode=AcquisitionType.CONTINUOUS)
+
+        actual_sampling_rate = task.timing.samp_clk_rate
+        print(f"Actual sampling rate: {actual_sampling_rate} Hz")
```

Comment:
Units should be S/s (samples/sec).

Use format 'f' or 'g' to eliminate excess precision. In this case, 'g' is fine because we don't need to display trailing zeros.

```suggestion
        print(f"Actual sampling rate: {actual_sampling_rate:g} S/s")
```

Reply:
all 3 cont examples are updated as per suggestion.